config/routes.rb

```diff
@@ -1,7 +1,9 @@
 Rails.application.routes.draw do
-  get 'home/index'
+  get '/product/', to: "product#index"
+  get '/home/', to: "home#index"
+  get 'product/:id', to: "product#show"
```

Comment:
@Krutik31 There is no need to create manual routes for product. You can write it like this
  resources :product, only: [:index, :show]

Reply:
Changes made on routes

Reply:
@Krutik31  in routes controller name are always plural